cmd/kwild/server/build.go

```diff
@@ -441,31 +452,52 @@ func initAccountRepository(d *coreDependencies, tx sql.Tx) {
 	}
 }
 
-func buildSnapshotter() *snapshots.SnapshotStore {
-	return nil
-	// TODO: Uncomment when we have statesync ready
-	// cfg := d.cfg.AppCfg
-	// if !cfg.SnapshotConfig.Enabled {
-	// 	return nil
-	// }
+func buildSnapshotter(d *coreDependencies) *snapshots.SnapshotStore {
+	cfg := d.cfg.AppCfg
+	if !cfg.Snapshots.Enabled {
+		return nil
+	}
 
-	// return snapshots.NewSnapshotStore(cfg.SqliteFilePath,
-	// 	cfg.SnapshotConfig.SnapshotDir,
-	// 	cfg.SnapshotConfig.RecurringHeight,
-	// 	cfg.SnapshotConfig.MaxSnapshots,
-	// 	snapshots.WithLogger(*d.log.Named("snapshotStore")),
-	// )
+	dbCfg := &snapshots.DBConfig{
+		DBUser: cfg.DBUser,
+		DBPass: cfg.DBPass,
+		DBHost: cfg.DBHost,
+		DBPort: cfg.DBPort,
```

Comment:
You need to also include db name here. I noticed you have it hardcoded in the pgDumpCmd to `kwild`, which is our default, however it is configurable.